stopcovid/event_distributor/outbound_sms.py

```diff
@@ -0,0 +1,106 @@
+import logging
+from typing import List
+from dataclasses import dataclass
+
+from stopcovid.dialog.types import DialogEvent
+from stopcovid.dialog.dialog import (
+    DrillCompleted,
+    AdvancedToNextPrompt,
+    FailedPrompt,
+    CompletedPrompt,
+    UserValidated,
+    DrillStarted,
+    UserValidationFailed,
+)
+from stopcovid.drills.localize import localize
+
+
+TRY_AGAIN = "{{incorrect_answer}}"
+
+USER_VALIDATION_FAILED_COPY = (
+    "Invalid Code. Check with your administrator and make sure you have the right code."
+)
+USER_VALIDATED_COPY = "You're all set! You should get your first drill soon."
+
+# We should template these for localization
+DRILL_COMPLETED_COPY = "Drill complete!"
+CORRECT_ANSWER_COPY = "Correct!"
```

Comment:
oh, I was using `{{right}}` in the simulator and then Kai's most recent update removed it.

@kyen99 what string should we use here?